standard/attributes.md

```diff
@@ -512,12 +550,14 @@ A method decorated with the `Conditional` attribute is a conditional method. Eac
 > *Example*:
 > ```csharp
 > using System.Diagnostics;
+>
 > class Eg
 > {
 >     [Conditional("ALPHA")]
 >     [Conditional("BETA")]
->     public static void M() {
->     //...
+>     public static void M()
+>     {
+>         //...
```

Comment:
Add a space between the `//` and `...`? A pet peeve of mine is comments written without capitals and without the space ;)